client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import {
+	camelCase,
+	difference,
+	isEmpty,
+	keys,
+	map,
+	pick,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getContactDetailsExtraCache } from 'state/selectors';
+import { updateContactDetailsCache } from 'state/domains/management/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+const legalTypes = {
```

Comment:
Not a blocker, but these should be translated to French (and not to any other langauges).
There's a list in [here](https://acei.ca/exigences-en-matière-de-présence-au-canada-applicable-aux-titulaires) or [pdf](https://acei.ca/sites/default/files/public/policy/cprregistrants-fr.pdf).
The question is if we want to just create an array of translations and use a custom translation functions, or use our regular translation workflow.
I'm leaning towards custom - but don't have a strong opinion.